core/dbt/include/global_project/macros/materializations/models/materialized_view/get_materialized_view_configuration_changes.sql

```diff
@@ -0,0 +1,23 @@
+{% macro get_materialized_view_configuration_changes(existing_relation, new_config) %}
```

Comment:
What do we do with the rest of NodeConfig?

Reply:
We don't actually track anything else at the relation level for materialized views in Postgres. It looks pretty close to table. We have not implemented contracts or constraints yet for materialized views, otherwise it's the same as table. However, I did refactor how I'm determining the differences so that it can be more extensible for the other adapters, or when we add functionality to materialized views.